docs/datamaps/index.md

```diff
@@ -0,0 +1,169 @@
+# Data Maps
+
+A registry data map contains data-driven, reloadable objects that can be attached to a registry object.  
+This system allows more easily data-driving game behaviour, as they provide functionality such as syncing or conflict resolution, leading to a better and more configurable user experience.  
+
+You can think of tags as entry->boolean maps, while data maps are more flexible entry->object maps.
```

Comment:
How is a tag an entry-> boolean? Or is entry the registry object in this case?

Reply:
Entry is the registry object, yes.